services/nyx/clang.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# resolve current clang toolchain
+resolve-toolchain () {
```

Comment:
Do you want to update this to use the changes in #327?

Reply:
Nevermind.  It's probably more trouble than it's worth to make both functions behave the same.